### PR TITLE
Kor 98 level manager

### PR DESCRIPTION
**Pull request by AlonLivne**
Turned LevelEditor into level manager, added level 1 and level 2 in progress

### CODE REVIEW

File: Assets/Scripts/LevelManager.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            return null;
+        }
+
+        if (isEnemy)
+        {
+            return GetValidCard(index, EnemyCards);
+        }
+        else
+        {
+            return GetValidCard(index, EnemyCards);
```
Comment (yoavhops) [inline Assets/Scripts/LevelManager.cs]: is this suppose to be "EnemyCards" here?

File: Assets/Scripts/LevelManager.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    public void InitCardsLists()
+    {
+        foreach (var levelMediator in LevelCardsMediators)
+        {
+            if (Level == levelMediator.Level)
+            {
+                HandCards = levelMediator.HandCards;
+                EnemyCards = levelMediator.EnemyCards;
```
Comment (yoavhops) [inline Assets/Scripts/LevelManager.cs]: Cool trick!

File: Assets/Scripts/LevelManager.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                return;
+            }
+        }
+        throw new NotImplementedException();
+    }
+
+    void Awake()
+    {
+        Singleton = this;
+        InitCardsLists();
```
Comment (yoavhops) [inline Assets/Scripts/LevelManager.cs]: nice

File: Assets/Scripts/LevelManager.cs (hunk 1 above)
Comment (AlonLivne) [reply Assets/Scripts/LevelManager.cs]: No, supposed to be HandCards

**Pull request merged**